core/src/main/resources/schema/infinispan-config-9.0.xsd

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified"
-           version="1.0" targetNamespace="urn:infinispan:config:8.2"
-           xmlns:tns="urn:infinispan:config:8.2" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+           version="1.0" targetNamespace="urn:infinispan:config:9.0"
```

Comment:
Sounds like we need to branch then :)  Unless we aren't going to do anything more for 8.2.